h2o-core/src/main/java/hex/ModelMojoWriter.java

```diff
@@ -148,7 +148,7 @@ protected final void finishWritingTextFile() throws IOException {
 
   private void addCommonModelInfo() throws IOException {
     int n_categoricals = 0;
-    for (String[] domain : model._output._domains)
+    for (String[] domain : model.scoringDomains())
```

Comment:
I am not sure what is the motivation behind the specific implementation of GLMModel.scoringDomains().

I wanted to maintain compatibility with POJOs. The function responsible for generating domains for POJOs calls this functions instead of directly accessing the field _output._domains.

Perhaps @tomasnykodym could explain why GLM overrides the default.